src/amp.js

```diff
@@ -178,6 +178,9 @@ if (shouldMainBootstrapRun) {
       self.location.href
     );
   }
+  if (getMode().test) {
```

Comment:
This gets DCE'd from production builds right?

Reply:
it does not. i am adding a `FORTESTING` symbol that will get eliminated.

Reply:
switched to `getMode().localDev` which is stripped out by a babel plugin